app/src/main/java/org/simple/clinic/enterotp/EnterOtpScreen.kt

```diff
@@ -75,7 +72,7 @@ class EnterOtpScreen(context: Context, attributeSet: AttributeSet) : RelativeLay
   fun showUserPhoneNumber(phoneNumber: String) {
     val phoneNumberWithCountryCode = resources.getString(
         R.string.enterotp_phonenumber,
-        resources.getString(R.string.country_dialing_code),
```

Comment:
Did we also remove this string resource?

Reply:
Yes, in a later commit.